index.js

```diff
@@ -561,7 +755,7 @@ module.exports = {
                     'q-applicant-when-did-the-crime-stop': '2020-01-10T00:00:00.000Z'
                 },
                 {
-                    'q-applicant-when-did-the-crime-stop': '2010-01-01T00:00:00.000Z'
+                    'q-applicant-when-did-the-crime-stop': '2010-01-10T00:00:00.000Z'
```

Comment:
Can we avoid unnecessary changes? With dates especially it's not immediately obvious if this is a breaking change or not.